app/src/main/kotlin/com/wire/android/model/ImageAsset.kt

```diff
@@ -44,6 +45,17 @@ sealed class ImageAsset(private val imageLoader: WireSessionImageLoader) {
             get() = userAssetId.toString()
     }
 
+    @Stable
+    data class LocalImageAsset(
```

Comment:
maybe some short comment wont hurt explaining what it is ? Without ready any further the code I do not have clear idea what this may be doing